stable/kafka-manager/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1beta1
```

Comment:
We offically support the current and previous minor versions of Kubernetes. Currently that would be 1.8+. You should use `apps/v1beta2`.